ui/pages/onboarding-flow/privacy-settings/privacy-settings.js

```diff
@@ -232,7 +231,6 @@ export default function PrivacySettings() {
                         e.preventDefault();
                         dispatch(showModal({ name: 'ONBOARDING_ADD_NETWORK' }));
                       }}
-                      icon={<Icon name="add" marginRight={2} />}
```

Comment:
## After
The icon for this Button wasn't showing up in `develop` but we don't really do icons in Buttons anyway so removing for consistency

<img width="972" alt="Screenshot 2023-05-19 at 2 38 12 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/70676338-b4b9-4969-9335-4c6c1a308366">